src/app/components/scan-fingers/scan-fingers.component.html

```diff
@@ -1 +1,37 @@
-<p>scan-fingers works!</p>
+<div
+  class="scanner__wrapper"
+  [ngStyle]="{
+    'border-color': !isReaderConnected
```

Comment:
please move the checker inside the component instead

Reply:
@hoangngk Updated